test/evergreen.yml

```diff
@@ -297,9 +308,8 @@ functions:
         else
           echo "Using config flags $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|}"
           # Fetch the gperftools library if needed. This will also get tcmalloc.
-          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]] && [ ! -d "./automation-scripts" ]; then
+          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]]; then
             is_cmake_build=true
-            git clone git@github.com:wiredtiger/automation-scripts.git
```

Comment:
Hope to point out there's [another PR](https://github.com/wiredtiger/wiredtiger/pull/10419/files#diff-42f20e21217bd7d4b14489056e1ea24ea2b87599c85413d49182a3b0b721c132) that's tasked to redesign tcmalloc a bit by removing the dependency to automation-scripts. So the changes here will not be necessary after that PR gets merged.

Reply:
Given Marc's struggles with that PR I might just proceed with the changes, the rebase for him should be trivial.